tests/test_caching.py

```diff
@@ -20,6 +20,14 @@ def test_xdg_config(self):
         finally:
             shutil.rmtree(tmpdir)
 
+    def test_cache_dir(self):
+        tmpdir = tempfile.mkdtemp()
+        try:
+            cache = memestra.caching.Cache(cache_dir=tmpdir)
```

Comment:
The assert will always succeed because `mkdtemp` creates the directory. You should probably call memestra using that cache, and check that a new file has been created :-)